Makefile

```diff
@@ -1,10 +1,15 @@
-UNITS = "@unit.abijson or @unit.algod or @unit.applications or @unit.atomic_transaction_composer or @unit.dryrun or @unit.feetest or @unit.indexer or @unit.indexer.logs or @unit.offline or @unit.rekey or @unit.transactions.keyreg or @unit.responses or @unit.responses.231 or @unit.tealsign or @unit.transactions or @unit.transactions.payment"
```

Comment:
Temporarily commented out. Indexer tests were failing on me (should be completely independent to my changes), but this issue needs to be resolved before merging.

Reply:
@tzaffi In case helpful:  I _think_ https://github.com/algorand/py-algorand-sdk/pull/301/ may fix the issue you faced.  I see the branch does _not_ contain the fix provided by https://github.com/algorand/py-algorand-sdk/pull/301/.

So, it's possible merging in `develop` fixes the problem.

Reply:
that seems to have done the trick. 🙏